src/pudl_archiver/depositors/zenodo.py

```diff
@@ -122,26 +127,52 @@ async def create_deposition(self, metadata: DepositionMetadata) -> Deposition:
         Returns:
             The brand new deposition - unpublished and with no files in it.
         """
-        url = f"{self.api_root}/deposit/depositions"
+        url = f"{self.api_root}/records"
         headers = {
+            "Accept": "application/vnd.inveniordm.v1+json",
             "Content-Type": "application/json",
         } | self.auth_write
 
-        payload = {
-            "metadata": metadata.dict(
-                by_alias=True,
-                exclude={"publication_date", "doi", "prereserve_doi"},
-            )
-        }
-
+        # Set up metadata from data source
+        metadata = metadata.dict(
+            by_alias=True,
+            exclude={"publication_date", "doi", "prereserve_doi"},
+        )
+        metadata["publication_date"] = str(date.today())
+
+        payload = json.dumps(
+            {
+                "access": {
+                    "record": "public",
+                    "files": "public",
+                },
+                "files": {"enabled": True},  # Must always be True for Zenodo
```

Comment:
It throws an error if True, not documented anywhere. As far as I can tell it's technically a setting that the network configurator can change to allow metadata only records but this doesn't seem to be enabled in Zenodo. This is ok since we only ever want to upload records with files. It also matches the behavior of old Zenodo API (see the expected failure for the `empty_deposition` test, e.g.